deploy/script/katello-backup

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Echo all commands
+usage(){
+  echo "Usage: katello-backup <backup directory>"
+  echo " eg: $ katello-backup /var/tmp/katello-backup/"
+}
+
+if [ "$#" -ne 1 ]; then
+  echo "ERROR: Please specify an export directory"
+  usage
+  exit 1
+fi
+
+# Backup dir
+umask 0027
+export BDIR=$1
+mkdir $BDIR
```

Comment:
Should we play it safe and add the `-p` flag to `mkdir` here? More than likely the user will use `/var/tmp/something` but it's not guaranteed. Calling `mkdir $HOME/parent/child/` when `parent` doesn't exist will error. `-p` tells `mkdir` to create all parent directories as necessary.